ansible/roles/prep.dirs/files/ceph-el8-repo.tpl.j2

```diff
@@ -0,0 +1,13 @@
+config_opts['yum.conf'] += """
+
+[ceph-repo]
+name = Ceph builds
+baseurl={{ ceph_el8_repo_base_url }}/x86_64
```

Comment:
should this be `ceph_base_url`?

Reply:
This is right, `ceph_el8_repo_base_url` gets set to `ceph_base_url` in prep.dirs/tasks/repo.yml if it's provided, or the appropriate url from shaman otherwise. `ceph_base_url` acts as an override for all versions.